packages/js/components/src/table/index.tsx

```diff
@@ -0,0 +1,250 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import classnames from 'classnames';
+import { createElement, Fragment, useState } from '@wordpress/element';
+import { find, first, without } from 'lodash';
+import React from 'react';
+import {
+	Card,
+	CardBody,
+	CardFooter,
+	CardHeader,
+	// @ts-expect-error: Suppressing Module '"@wordpress/components"' has no exported member '__experimentalText'
+	__experimentalText as Text,
+} from '@wordpress/components';
```

Comment:
I'm not sure why TS complains about this import 🤔  It is exported [here](https://github.com/WordPress/gutenberg/blob/297565c13e84b07d17b9d08b1066927190a9f2bd/packages/components/src/index.js#L147)

Reply:
I think it's because it is not exported in [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/wordpress__components/index.d.ts).

Reply:
Ah that makes sense @chihsuan 👍 Thank you for pointing it out!